Description/Enhancer/Sylius/ResourceEnhancer.php

```diff
@@ -81,14 +82,38 @@ public function enhance(Description $description)
         ];
 
         foreach ($map as $descriptor => $action) {
+            try {
+                $url = $this->urlGenerator->generate(
+                    $configuration->getRouteName($action),
+                    [
+                        'id' => $payload->getId(),
+                    ]
+                );
+                $description->set($descriptor, $url);
+            } catch (RouteNotFoundException $e) {
```

Comment:
not sure if its that bad. it could hide issues though, if i misconfigured something and then i will just not get a link instead of an exception that makes me notice i mistyped a name somewhere.

could configuration->getRouteName decide if there is a route or not, so that you could do an `if` here?